src/StreamJsonRpc.Tests/JsonRpcTests.cs

```diff
@@ -119,19 +119,52 @@ public void Attach_Null_Throws()
         Assert.Equal(testLine + "!", result1);
     }
 
+    [Fact]
+    public async Task CanInvokeMethodOnServer_Overload()
+    {
+        var streams = FullDuplexStream.CreateStreams();
+        var overloadRpc = new JsonRpcCrashesOnException(streams.Item1, streams.Item2, new Server());
+        overloadRpc.StartListening();
```

Comment:
Given the number of tests you added and the proportion of each one containing this boilerplate code, IMO this should be moved into its own test class with a constructor that contains the boilerplate code.

Reply:
Also, it appears you've doubled up on a lot of existing test methods. If it's necessary to achieve double-coverage for the different configuration, let's do that in a way that doesn't require maintaining double copies of every test. But I'm hoping that we only need to double-up on the tests that actually include the server throwing exceptions.
BTW, I don't see any code here to handle or suppress crashing. Where is it?